src/test/java/org/broadinstitute/hellbender/tools/examples/ExampleTwoPassVariantWalkerIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.examples;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.FeatureDataSource;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+
+import static org.broadinstitute.hellbender.tools.examples.ExampleTwoPassVariantWalker.COPY_OF_QD_KEY_NAME;
+import static org.broadinstitute.hellbender.tools.examples.ExampleTwoPassVariantWalker.QD_DISTANCE_FROM_MEAN;
+import static org.broadinstitute.hellbender.utils.variant.GATKVCFConstants.QUAL_BY_DEPTH_KEY;
+
+public class ExampleTwoPassVariantWalkerIntegrationTest extends CommandLineProgramTest {
+    @Test
+    public void test() throws IOException {
+        final File outputVcf = File.createTempFile("output", "vcf");
+        final String inputVcf = "src/test/resources/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants/haploid-multisample.vcf";
+
+        final String[] args = {
+                "-" + StandardArgumentDefinitions.VARIANT_SHORT_NAME, inputVcf,
+                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputVcf.getAbsolutePath()
+        };
+
+        runCommandLine(args);
+
+        final FeatureDataSource<VariantContext> variantsBefore = new FeatureDataSource<>(inputVcf);
+        final FeatureDataSource<VariantContext> variantsAfter = new FeatureDataSource<>(outputVcf);
```

Comment:
Create these data sources in a try-with-resources block, to ensure that they get closed.

Reply:
done